src/test/run-make-fulldeps/hotplug_codegen_backend/the_backend.rs

```diff
@@ -44,10 +45,10 @@ impl CodegenBackend for TheBackend {
         &self,
         tcx: TyCtxt<'a, 'tcx, 'tcx>,
         _rx: mpsc::Receiver<Box<Any + Send>>
-    ) -> Box<Any> {
```

Comment:
This test file was checking that you could implement a codegen backend via a trait. I had changed the trait signature in order to collect the `DefId`s that need to appear in the MIR ELF section, so we need to change the test.